integration-tests/devtools/src/test/java/io/quarkus/devtools/commands/AddGradleExtensionsTest.java

```diff
@@ -71,18 +70,26 @@ public TestingGradleBuildFile(Path projectDirPath, QuarkusPlatformDescriptor pla
         }
 
         @Override
-        protected List<Dependency> getDependencies() throws IOException {
+        protected List<AppArtifactCoords> getDependencies() throws IOException {
             final Matcher matcher = Pattern.compile("\\s*implementation\\s+'([^\\v:]+):([^\\v:]+)(:[^:\\v]+)?'")
                     .matcher(getBuildContent());
-            final ArrayList<Dependency> builder = new ArrayList<>();
+            final ArrayList<AppArtifactCoords> builder = new ArrayList<>();
             while (matcher.find()) {
-                final Dependency dep = new Dependency();
-                dep.setGroupId(matcher.group(1));
-                dep.setArtifactId(matcher.group(2));
-                dep.setVersion(matcher.group(3));
-                builder.add(dep);
+                builder.add(createDependency(matcher.group(1), matcher.group(2), matcher.group(3), "jar"));
+            }
+            if (getBuildContent().contains(
+                    "implementation enforcedPlatform(\"${quarkusPlatformGroupId}:${quarkusPlatformArtifactId}:${quarkusPlatformVersion}\")")) {
```

Comment:
We need to make this less brittle in the future (maybe regexp?)

Reply:
In the future it should be checking for any specific property names at all.

Reply:
Ah, it's a test, sorry. Ignore my comment above.

Reply:
@gastaldi this is just for testing ;-)